src/dsp_tools/cli/entry_point.py

```diff
@@ -56,6 +55,7 @@ def run(args: list[str]) -> None:
         user_args=args,
         parser=parser,
     )
+    _check_version()
```

Comment:
I moved this down, because otherwise, you are even prompted if you just want to print the help message (i.e. just type `dsp-tools`, then return).